frontend/app/src/components/dashboard/edit-snapshot/EditBalancesSnapshotAssetPriceForm.vue

```diff
@@ -0,0 +1,313 @@
+<script setup lang="ts">
+import { toMessages } from '@/utils/validation';
+import { CURRENCY_USD } from '@/types/currencies';
+import { TaskType } from '@/types/task-type';
+import { ApiValidationError, type ValidationErrors } from '@/types/api/errors';
+import type { Validation } from '@vuelidate/core';
+import type { BigNumber } from '@rotki/common';
+import type { HistoricalPriceFormPayload } from '@/types/prices';
+import type { NewHistoryEventPayload } from '@/types/history/events';
+import type { ActionStatus } from '@/types/action';
+
+const props = withDefaults(
+  defineProps<{
+    timestamp: number;
+    asset: string;
+    disableAsset?: boolean;
+    v$: Validation;
```

Comment:
Is this the proper way to do this by passing the instance?

There is this entry that talks about [nested validations](https://vuelidate-next.netlify.app/advanced_usage.html#nested-validations) that doesn't seem to pass the instance. Can you have a look to see if this is possible?

Reply:
Im just need the error message from the `v$`.

Reply:
Yes this happens because the validation for this sub-form is external to the component. My question is, do we have any specific reason that blocks us from moving the relevant parts of the validation to the component itself instead of passing it externally?

Do we gain something by passing the validation instead of specifying it inside the component?

Reply:
I see, yes I think this can be moved to the children instead, let me move it